### PR TITLE
Updated icu to 69.1 and migrated it to submodule

### PR DESCRIPTION
**Pull request by biodranik**
Please test if it fixes RTL text issues and add corresponding issue references.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -32,3 +32,6 @@
 [submodule "3party/glm"]
   path = 3party/glm
   url = https://github.com/g-truc/glm.git
+[submodule "3party/icu/icu"]
+	path = 3party/icu/icu
```
Comment (rtsisyk) [inline .gitmodules:36]: Nit: please use space as for other libraries.

File: coding/transliteration.cpp [hunk 2]
```diff
@@ -5,11 +5,12 @@
 #include "base/logging.hpp"
 #include "base/string_utils.hpp"
 
-#include "3party/icu/common/unicode/uclean.h"
-#include "3party/icu/common/unicode/unistr.h"
-#include "3party/icu/common/unicode/utypes.h"
-#include "3party/icu/i18n/unicode/translit.h"
-#include "3party/icu/i18n/unicode/utrans.h"
+#include "unicode/putil.h"
```
Comment (rtsisyk) [inline coding/transliteration.cpp:8]: `<>` instead of "" ?

File: xcode/coding/coding.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -838,14 +838,15 @@
 				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
 				GCC_PREPROCESSOR_DEFINITIONS = (
 					"$(inherited)",
-					"U_DISABLE_RENAMING=1",
+					UCONFIG_USE_LOCAL,
```
Comment (rtsisyk) [inline xcode/coding/coding.xcodeproj/project.pbxproj:841]: What is it?

**Review by rtsisyk — APPROVED** [review]
Both Android && iOS works OK.

**Comment by rtsisyk** [comment]
Unit tests are failing. Please fix them before merging.

**Review by vng — APPROVED** [review]

**Review by vng — CHANGES_REQUESTED** [review]
Транслитерация почему-то не завелась:

Running transliteration_test.cpp::Transliteration_CompareSamples
Cannot create transliterator " Any-Latin;NFD;[ʹ-˓́-͘·']Remove;NFC ", icu error = 65569
FAILED
map_tests/transliteration_test.cpp:15 TEST(expected == out) alrbyt العربية

File: xcode/coding/coding.xcodeproj/project.pbxproj (hunk 3 above)
Comment (biodranik) [reply xcode/coding/coding.xcodeproj/project.pbxproj:841]: Forces include of our own configuration file `3party/icu/uconfig_local.h`
See https://unicode-org.github.io/icu-docs/apidoc/released/icu4c/uconfig_8h_source.html#l00054

**Comment by biodranik** [comment]
Fixed all comments. PTAL.

**Comment by rtsisyk** [comment]
LGTM. Let's wait for @vng and tests.

**Review by vng — APPROVED** [review]

**Pull request merged**